CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Dependencies
 - Bump `actions/checkout` from 3 to 4 ([#187](https://github.com/Cray-HPE/csm-config/pull/187))
 
+## [1.16.19] - 2023-09-14
```

Comment:
Need to update the version links at the bottom of the file for this new version number (specifically, add an entry for this new version, and then update the unreleased version link)